datameta/templates/layout.pt

```diff
@@ -40,20 +40,77 @@
         </style>
         <metal:macro metal:define-slot="head"/>
         <script>(function(e,t,n){var r=e.querySelectorAll("html")[0];r.className=r.className.replace(/(^|\s)no-js(\s|$)/,"$1js$2")})(document,window,0);</script>
+        <script>
+        function getCookie(cname) {
```

Comment:
I think for your particular cookie you can keep obtaining the cookie as simple as
```javascript
document.cookie.split("; ").find(row=>row.startsWith("widescreen=")).split('=')[1]
```
* Since we can ensure the values are either `true` or `false`, we don't have to worry about percent-encoding
* The RFC requires a single space after the semicolon.